src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { FC, ReactNode, ReactElement } from 'react';
+import { GuidanceState, DocumentationResource, State } from 'pathways-model';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import styles from './ExpandedNode.module.scss';
+import indexStyles from 'styles/index.module.scss';
+import { isGuidanceState, isBranchState } from 'utils/nodeUtils';
+
+interface ExpandedNodeProps {
+  pathwayState: GuidanceState | State;
+  isActionable: boolean;
+  documentation: DocumentationResource | undefined;
+}
+
+const ExpandedNode: FC<ExpandedNodeProps> = ({ pathwayState, isActionable, documentation }) => {
+  const guidance =
+    isGuidanceState(pathwayState) && renderGuidance(pathwayState as GuidanceState, documentation);
+  const branch = isBranchState(pathwayState) && renderBranch(documentation);
+
+  return (
+    <div className={indexStyles.expandedNode}>
+      <table className={styles.infoTable}>
+        <tbody>{guidance || branch}</tbody>
+      </table>
+      {isActionable && (
+        <form className={styles.commentsForm}>
+          <label>Comments:</label>
+          <button className={indexStyles.button} onClick={(e): void => e.preventDefault()}>
```

Comment:
Looks like the default text functionality was removed and no longer works...

https://github.com/mcode/pathways/blob/f6bb95a7622b01ed4cfb06a39e27b0c6b8e8320f/src/components/DocNode/DocNode.tsx#L41-L42

This is one example of the dangers of changing the names of files and making a lot of other changes...

Reply:
Added back the default comments functionality. Also renamed ExpandedNode to DocNode for this PR